apps/server/src/modules/files-storage/mapper/files-storage.mapper.spec.ts

```diff
@@ -40,6 +40,10 @@ describe('FilesStorageMapper', () => {
 			const result = FilesStorageMapper.mapToAllowedAuthorizationEntityType(FileRecordParentType.User);
 			expect(result).toBe(AllowedAuthorizationEntityType.User);
 		});
+		it('should return allowed type equal Submission', () => {
```

Comment:
In this description block are no empty lines between the its. I think we can put one into it between each line.